package.json

```diff
@@ -26,7 +26,8 @@
     "cfn-response": "*",
     "moment": "^2.19.2",
     "request": "*",
-    "request-promise-native": "*"
+    "request-promise-native": "*",
+    "al-collector-js": "git://github.com/alertlogic/al-collector-js#master"
```

Comment:
Let's call the dependency as just `al-collector`.

Reply:
What about publishing the package in NPM https://www.npmjs.com/

Reply:
I think it is fine as we agreed not to do it unless it is really necessary for npm usage.